src/environmentd/src/http/sql.rs

```diff
@@ -177,7 +177,7 @@ async fn run_ws(state: &WsState, mut ws: WebSocket) {
         };
 
         if let Err(err) = ws_response().await {
-            tracing::error!("failed to send respond over WebSocket, {err:?}");
+            debug!("failed to send respond over WebSocket, {err:?}");
```

Comment:
If you happen to need to go back through CI

```suggestion
            debug!("failed to send response over WebSocket, {err:?}");
```